lib/daru/category.rb

```diff
@@ -0,0 +1,738 @@
+module Daru
+  module Category
```

Comment:
Rubocop complains about module having too many lines. Also I get this ABCSize metric:

```ruby
lib/daru/vector.rb:173:5: C: Metrics/AbcSize: Assignment Branch Condition size for initialize is too high. [21.77/20]
    def initialize source, opts={}
```

How do I solve these two?